rio/subsystems/yoke.py

```diff
@@ -40,11 +40,17 @@ def __init__(self) -> None:
             CANSparkMaxLowLevel.MotorType.kBrushless,
         )
 
+        # MOVE ME
+        self.starShooter.setInverted(True)
+
```

Comment:
Why are these here? We already confirmed motor direction for the starboard side, only the port is inverted?

Reply:
both of them are set to be inverted they're just set in different places below the PID that is being inverted